### PR TITLE
Mongo count emailer

### PR DESCRIPTION
**Pull request by drewjj**
# PR Details

## Description

Reworks the count_metric addon service to be used as a daily emailer that sends out the jpo-ode message counts ever 24 hours. This also includes a rework to streamline the count creation process by using the in-environment mongoDB database for the source of the counts. The emailer dynamically builds an HTML table for RSUs that are both in PostgreSQL and any unknown units that might show up in mongoDB.

It is now noted that the old counter portion of the service is now marked as deprecated. We will remove support for this in the future. The emailer cannot be used with the pre-existing counter.

## How Has This Been Tested?

This has been tested through unit tests, running portions of the new mongoDB queries manually and through the CDOT GCP-dev environment in K8s.

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [x] My changes require new environment variables:
  - [x] I have updated the docker-compose, K8s YAML, and all dependent deployment configuration files.
- [x] My changes require updates to the documentation:
  - [x] I have updated the documentation accordingly.
- [x] My changes require updates and/or additions to the unit tests:
  - [x] I have modified/added tests to cover my changes.
- [x] All existing tests pass.

### CODE REVIEW

**Review by dmccoystephenson — APPROVED** [review]
Code looks good & the services unit tests pass!

**Review by mwodahl — APPROVED** [review]
This looks good to me!

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Looks like we need to update the requirements.txt file in order for the unit tests to pass in our python 12 venv environments:

multidict==6.0.5

Currently targeting 6.0.4 but with this, unit tests fail

**Comment by drewjj** [comment]
> Looks like we need to update the requirements.txt file in order for the unit tests to pass in our python 12 venv environments:
> 
> multidict==6.0.5
> 
> Currently targeting 6.0.4 but with this, unit tests fail

This is resolved

File: services/addons/images/count_metric/daily_emailer.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+            rsu_dict[row["ipv4_address"]]["counts"][type] = {"in": 0, "out": 0}
+    logging.debug(f"Created RSU dictionary: {rsu_dict}")
+
+    return rsu_dict
+
+
+def email_daily_counts(email_body):
+    logging.info("Attempting to send the count emails...")
+    try:
+        email_addresses = os.environ["SMTP_EMAIL_RECIPIENTS"].split(",")
```
Comment (payneBrandon) [inline services/addons/images/count_metric/daily_emailer.py:130]: Not needed for this PR by any means, but what do you think about setting this as configurable from the CV Manager UI? Allow people to subscribe to this tool's email outputs via a new database setting? Then we'd have a simple location for managing/viewing who is getting these and we could add/remove as needed.

File: services/addons/images/count_metric/gen_email.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
+import logging
+import os
+import daily_emailer
```
Comment (payneBrandon) [inline services/addons/images/count_metric/gen_email.py]: We've got a circular reference here with `gen_email` importing `daily_emailer` and `daily_emailer` importing `get_email`. Could we get around this by removing this reference here and instead passing in the values that we want to iterate over via method parameter instead?

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just a couple comments, looking good!

File: services/addons/images/count_metric/daily_emailer.py (hunk 1 above)
Comment (drewjj) [reply services/addons/images/count_metric/daily_emailer.py:130]: Yeah that could be pretty neat actually. I will create a backlog ticket for that.

File: services/addons/images/count_metric/gen_email.py (hunk 2 above)
Comment (drewjj) [reply services/addons/images/count_metric/gen_email.py]: Removed the circular import. Interesting that I never had an issue with this during testing.

**Review by mwodahl — APPROVED** [review]
These changes look good to me!

File: services/addons/images/count_metric/gen_email.py (hunk 2 above)
Comment (payneBrandon) [reply services/addons/images/count_metric/gen_email.py]: Python will happily let you create a circular dependency, but runtime can have odd repercussions

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**